<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>planificar</class>
 <widget class="QDialog" name="planificar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="hora">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hora</string>
   </property>
  </widget>
  <widget class="QLabel" name="actividad">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Actividad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonagregar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipoact">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de actividad </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="lun">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>86</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Lun</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>86</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mier">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>86</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mie</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="jue">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>86</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Jue</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vie">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>86</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Vie</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sab">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>86</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sab</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dom">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>86</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Dom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hora_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="actividad_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipoact_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
